### ISSUE TITLE
請問有辦法把之前的mediawiki的頁面都轉過來嗎？ 

### ISSUE THREAD

**Issue by changchichung** [tankywoo/simiki#108]
很喜歡這個 simiki 感謝你的付出
我們之前都是用mediawiki，請問有辦法把 mediawiki上面的頁面都轉過來 simiki嗎？

**Comment by Hsins** [comment]
You should know that Simiki is a static page generator. All the HTML files are generated from the markdown file.

Therefore, the crux of the matter is that you should turn your mediawiki pages into the markdown files.

**Comment by changchichung** [comment]
yup , I realize that  , I will try to convert all pages using pandoc , thanks !

**Comment by tankywoo** [comment]
需要转换为 markdown 格式，并且需加上 [页面头信息](http://simiki.org/zh-docs/metadata.html)
